### PR TITLE
Remove some pi-gen leftovers

### PR DESCRIPTION
**Pull request by AaronDewes**
This makes Umbrel OS more different from Raspberry Pi OS

### CODE REVIEW

**Comment by mayankchhabra** [comment]
Looks great, thanks @AaronDewes! Will test the build.

**Comment by mayankchhabra** [comment]
Hey @AaronDewes, how about we just add the following to the `config` file?

```
PI_GEN="Umbrel OS official build scripts"
PI_GEN_REPO="https://github.com/getumbrel/umbrel-os"
```

**Comment by AaronDewes** [comment]
Is this okay?

**Comment by AaronDewes** [comment]
Sorry, rebased wrong, I'll fix that.

**Comment by AaronDewes** [comment]
Fixed. Image will be at https://github.com/getumbrel/umbrel-os/actions/runs/187247427

**Comment by AaronDewes** [comment]
I'll rebase this after PR #84 is merged, rebasing now would just result in another conflict with this PR.

File: export-image/04-finalise/01-run.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ if [ -d "${ROOTFS_DIR}/home/${FIRST_USER_NAME}/.config" ]; then
 fi
 
 rm -f "${ROOTFS_DIR}/etc/apt/apt.conf.d/51cache"
-rm -f "${ROOTFS_DIR}/usr/bin/qemu-arm-static"
+rm -f "${ROOTFS_DIR}/usr/bin/qemu-aarch64-static"
```
Comment (AaronDewes) [inline export-image/04-finalise/01-run.sh:20]: This is a fix for something I forgot in the arm64 port.

**Comment by AaronDewes** [comment]
I rebased it.
The version now sets some default values to Umbrel values, removes one unused step, removes pi-gen references from issue.txt and renames rpi-issue to umbrel-issue.

**Comment by mayankchhabra** [comment]
Tested! Thanks, @AaronDewes. Looks great!

```
umbrel@umbrel:~ $ cat /etc/umbrel-issue
Umbrel OS built on 2020-08-10
Generated using Umbrel OS official build scripts, Umbrel OS official build scripts, 4828436ca980f00be30747e8df87be7edd91e36d, stage2
```

**Comment by mayankchhabra** [comment]
Ah, there're some conflicts. Could you please fix them? I'll go ahead and merge after that

**Comment by AaronDewes** [comment]
Done.

File: build.sh [hunk 2]
```diff
@@ -143,8 +143,8 @@ do
 	esac
 done
 
-export PI_GEN=${PI_GEN:-pi-gen}
-export PI_GEN_REPO=${PI_GEN_REPO:-https://github.com/RPi-Distro/pi-gen}
+export BUILD_SCRIPTS="${BUILD_SCRIPTS:-Umbrel OS official build scripts}"
+export BUILD_SCRIPTS_REPO="${BUILD_SCRIPTS:-https://github.com/getumbrel/umbrel-os}"
```
Comment (mayankchhabra) [inline build.sh:147]: I think you meant `BUILD_SCRIPTS_REPO` here

```suggestion
export BUILD_SCRIPTS_REPO="${BUILD_SCRIPTS_REPO:-https://github.com/getumbrel/umbrel-os}"
```

Comment (AaronDewes) [reply build.sh]: Yes, thank you!

**Pull request merged**